src/config/firmware.md

```diff
@@ -1,12 +1,12 @@
-# Microcode
+# Firmware
```

Comment:
I think we need to mention that some firmware packages might only be available from nonfree repos.